Resources/Prototypes/_Goobstation/Voice/speech_emotes.yml

```
@@ -154,3 +156,63 @@
    - flaps their wings
    - flaps its wings
+
+- type: emote
+  id: Trill
+  name: chat-emote-name-trill
+  category: Vocal
+  available: false
+  icon: _Goobstation/Interface/Emotes/caw.png
+  whitelist:
+    components:
+      - Vocal
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)